### ISSUE TITLE
can't build caddy, variable mismatch

### ISSUE THREAD

**Issue by unixandria-xda** [caddyserver/xcaddy#34]
Using go 1.15, manually installed to /usr/local/go I get:

```
/home/user/go/pkg/mod/github.com/smallstep/certificates@v0.15.0-rc.1.0.20200506212953-e855707dc274/authority/provisioner/jwk.go:155:24: assignment mismatch: 3 variables but x509util.SplitSANs returns 4 values  
 
/home/user/go/pkg/mod/github.com/smallstep/certificates@v0.15.0-rc.1.0.20200506212953-e855707dc274/authority/provisioner/x5c.go:197:24: assignment mismatch: 3 variables but x509util.SplitSANs returns 4 values                                          

2020/08/31 13:23:25 [INFO] Cleaning up temporary folder: /tmp/buildenv_2020-08-31-1323.298003067                              

2020/08/31 13:23:25 [FATAL] exit status 2

/home/user/go/pkg/mod/github.com/smallstep/certificates@v0.15.0-rc.1.0.20200506212953-e855707dc274/authority/provisioner/jwk.go:155:24: assignment mismatch: 3 variables but x509util.SplitSANs returns 4 values                                         

 /home/user/go/pkg/mod/github.com/smallstep/certificates@v0.15.0-rc.1.0.20200506212953-e855707dc274/authority/provisioner/x5c.go:197:24: assignment mismatch: 3 variables but x509util.SplitSANs returns 4 values                                          

2020/08/31 13:23:25 [INFO] Cleaning up temporary folder: /tmp/buildenv_2020-08-31-1323.298003067                              

2020/08/31 13:23:25 [FATAL] exit status 2
```

The cmd was `  xcaddy build --with github.com/caddy-dns/cloudflare`

**Comment by RyanCasas** [comment]
See this issue: #30 

@mholt any actual fix for this? I'm unable to recompile Caddy v2 with the Cloudflare module, either with Go 1.14 or Go 1.15.

**Comment by mholt** [comment]
Make sure to build with the latest commits; this should be resolved after Caddy 2.2 is released. Go doesn't build with pre-releases by default though.

**Comment by unixandria-xda** [comment]
> Make sure to build with the latest commits; this should be resolved after Caddy 2.2 is released. Go doesn't build with pre-releases by default though.

Any idea if it'll be released in say the next 90 days? 
Cloudflare CDN doesn't play well with certain renewal with default methods and the cloudflare DNS module seems like just what I really need

**Comment by mholt** [comment]
Yeah. I'm hoping it'll be released in the next 7-14 days.

**Comment by RyanCasas** [comment]
@mholt I can confirm this has been solved with the release of Caddy 2.2. I've been able to build it with the CloudFlare module with Go 1.15.2. So I think we can close this issue.

**Comment by mholt** [comment]
Great! Thanks for following up.
